src/main/java/no/digipost/api/useragreements/client/ApiService.java

```diff
@@ -181,11 +181,11 @@ public DocumentContent getDocumentContent(final SenderId senderId, final Agreeme
 	}
 
 	private static String userAgreementsPath(final SenderId senderId) {
-		return senderId.serialize() + "/" + USER_AGREEMENTS_PATH;
+		return "/" + senderId.serialize() + "/" + USER_AGREEMENTS_PATH;
```

Comment:
Why this change? Has there been some issues where the path would not resolve correctly, or a change in uris from the api (i.e. the url this is appended to did not end with `/`) etc.?

Reply:
No change in the api, but while I was testing it turned out that not prepending a `/` to these paths could cause some problems when resolving the complete URI - when they start with a `/` they will always resolve correctly.

Reply:
I see. I was concerned of double forward slashes, but since you have tested it should be ok.